src/Application/index.js

```diff
@@ -33,7 +33,6 @@ if (window.innerHeight === 720) {
 }
 
 export default function(App, appData, platformSettings) {
-  Metrics.app.launch()
```

Comment:
@wouterlucas we are sure that we don't want to track the `app.launch` metric anymore? just confirming since we still do have `app.close` in the close method

Reply:
Oh! Close should be gone also, oversight on my end. Thanks for catching that.